### ISSUE TITLE
Unexpected behavior due to variable versions of dependencies

### ISSUE THREAD

**Issue by codedot** [ripple/ripple-lib#736]
The latest version 4.8.0 of the `ajv` package prints the following message to standard output

```
$ref: all keywords used in schema at path "#". It will change in the next major version, see issue #260. Use option { extendRefs: true } to keep current behaviour
```

when `.preparePayment()` method is called in RippleAPI.

Requesting freezing all dependencies of `ripple-lib` at exact versions tested.

One workaround is to specify

```
        "dependencies": {
                "ajv": "4.0.5",
                "ripple-lib": "0.17.3"
        }
```

in `package.json` of a package that requires `ripple-lib`.
##

**Comment by ihomp** [comment]
same for me, 

https://github.com/epoberezkin/ajv/issues/303
https://github.com/epoberezkin/ajv/issues/260

**Comment by ihomp** [comment]
can be closed, not an issue in 0.17.7
just update :)

**Comment by intelliot** [comment]
Thank you!
